### DISCUSSION TITLE
Write UID to Mifare Classic 1k

### DISCUSSION THREAD

**Discussion by afif-malghani** [ikarus23/MifareClassicTool#491] [General]
Hello everyone,

I am new to NFCs I am trying to write to a Mifare classic 1K and am unable to write to block 0 sector 0. Is there a specific type of card that supports this? If not, is there a way to force write?

**Comment by ArchangeGabriel** [comment]
Yes, you need specific “magic” card that have writable UID. Do note that they are different versions of them, gen1 ones require special commands that cannot be sent by MCT (so only work from a computer), and they are also some that can only be written once.

**Reply by afif-malghani** [reply]
It might be possible to get emulation using a little bit of kernel hacking, but I don't  wanna get into it. Maybe later. However, I did see a reddit comment that some chinese phones support UID emulation out of the box. That's probably modified kernel as well.
